doc/coq.rst

```diff
@@ -152,6 +152,18 @@ The semantics of the fields are:
   Previous versions of Dune before 3.7 would disable the native rules depending
   on whether or not the ``dev`` profile was selected.
 
+Coq Dependencies
+~~~~~~~~~~~~~~~~
+
+When a Coq file ``a.v`` depends on another file ``b.v``, Dune is able to build
+them in the correct order, even if they are in separate theories.
+
+Starting with Dune 3.7, the dependencies between the ``.v`` files are computed
+using ``coqdep`` by calling it once per theory, producing a
+``mytheory.theory.d`` file in "makefile format", for a theory named
+``mytheory``. Before Dune 3.7, ``coqdep`` was called once per file and this was
```

Comment:
```suggestion
``mytheory``. Before Dune 3.7, ``coqdep`` was called once per file, and it was
```

Reply:
> Starting with Dune 3.7, the dependencies between the ``.v`` files are computed
using ``coqdep`` by calling it once per theory, producing a
``mytheory.theory.d`` file in "makefile format", for a theory named
``mytheory``. Before Dune 3.7, ``coqdep`` was called once per file and this was...

Starting with Dune 3.7, compute the dependencies between the ``.v`` files by calling ``coqdep`` once per theory, which produces a ``mytheaory.theory.d`` file (for a theory named ``mytheory``) in "makefile" format. 


Is this better, @Alizter ?

Reply:
Alternatively,``coqdep`` was called once per file before Dune 3.7, which was noticeably slower.

Reply:
> Here is what I want to get accross:
> 
> * Starting with Dune 3.7
>   
>   * The dependencies between the `.v` files are computed using `coqdep`.
>   * This is done by calling it once per theory
>     
>     * For a theory named `mytheory`
>       
>       * This produces a `mytheory.theory.d` file in "makefile format".
> * Before Dune 3.7
>   
>   * `coqdep` was called once per file and this was noticably slower.
> 
> I am having trouble turning this into a readable paragraph.

See what I proposed above. I'm happy to try again if you still don't like it.